marimbabot_planning/src/actions/hit_sequence.action

```diff
@@ -0,0 +1,8 @@
+#goal definition
+int32[] sequence
```

Comment:
We should add a tempo to the goal

Reply:
E.g. one int with bmp

Reply:
For the representation discussion see #73. Imo. it could either be the lilypond string or a sequence of hit points with timings. MIDI onsets would also be an option, but it is not really used in our code currently. Either way, I think ints are not a suitable representation here.

Reply:
> MIDI onsets would also be an option, but it is not really used in our code currently

Maybe I formulated that wrongly when I mentioned it yesterday: You definitely don't want to use MIDI as interface format (although we even have [some level of support for that](https://github.com/TAMS-Group/tams_midi_publisher) from prior work). But if you send 1. timing 2. a [midi-note number](http://midi.teragonaudio.com/tech/midispec/noteon.htm) (corresponding to western musical notation half-tones around mid-C with value 60) and a "velocity" or "loudness" parameter for each hit, that is essentially the information stored in a Note-Ons of a midi-file as well :-)

Reply:
I implemented an int for bpm and list of strings for the notes